changelog.d/21486_windows_2022_build.feat.md

```diff
@@ -0,0 +1,6 @@
+The Vector CI now uses
```

Comment:
I think this sort of CI change is likely to just be noise to end-users (i.e. they won't care). Vector developers may care, but the changelog audience is end-users. I'd probably mark this as `no-changelog`.